clarifai/cli/base.py

```diff
@@ -49,9 +80,16 @@ def login(ctx, config, env, user_id):
   if user_id:
     ctx.obj['user_id'] = user_id
     os.environ["CLARIFAI_USER_ID"] = ctx.obj['user_id']
-  elif 'user_id' in ctx.obj or 'CLARIFAI_USER_ID' in os.environ:
-    ctx.obj['user_id'] = ctx.obj.get('user_id', os.environ["CLARIFAI_USER_ID"])
+  elif 'user_id' in ctx.obj:
+    ctx.obj['user_id'] = ctx.obj.get('user_id', "")
     os.environ["CLARIFAI_USER_ID"] = ctx.obj['user_id']
+  elif 'CLARIFAI_USER_ID' in os.environ:
+    ctx.obj['user_id'] = os.environ["CLARIFAI_USER_ID"]
+  else:
+    user_id = click.prompt("Pass the User ID here", type=str)
+    os.environ["CLARIFAI_USER_ID"] = user_id
```

Comment:
I'm not sure about this too, because might not be useful for other CLI command other than CO related like model upload, and predict, etc. But let keep this for now, will update latter if needed

Reply:
Ok